_posts/2016-10-31-Kickstarters-Project.md

```diff
@@ -0,0 +1,205 @@
+During the first month they received different courses in which they could improve their technical skills about:
+
+ * Back-end
+    * Java (7 & 8) + JavaEE
+    * Spring
+    * JPA
+    * Webservices (REST & SOAP)
+    * MongoDB
+
+ * Front-end
+    * HTML & CSS
+    * Javascript & Typescript
+    * Angular
+    * Ionic
+
+These courses were given by the JWorks unit who tried to teach the kickstarters a much as possible with theoretical material and some exercises afterwards.
+Unfortunately these technical skills aren't enough to survive in the forever changing IT world.
+This is why some extra help was provided in the shape of books and courses about how to write clean code, how to work agile and learning how to work in a team while understanding and using the SCRUM principles.
+
+So by paying attention to technical development with the necessary certification processes and also focussing on the development of soft skills like communication, advising and collaboration,
+Ordina commits to the personal development of these kickstarters.
+
+
+
+## September - Dev-case
+
+The focus during the second month was on the implementation of this year's dev-case.
+Although they still had to follow a few courses along the way, like the basic priciples of security, the use of GIT and learning how to use MongoDB.
+
+### SensyMcSenseFace was born
+-- <cite>Chosen by popular vote, who would have guessed it...</cite></br>
+
+The kickstarters already had learned how to write clean code and how to do this in the best possible way.
+The purpose of the SensyMcSenseFace project was to give the kickstarters a use case where they could develop an end-to-end IoT solution, in which they could test and use their newly acquired skills.
+
+<p style="text-align: center;">
+  <img alt="SensyMcSenseFaceProject" src="/img/IoT.jpg">
+</p>
+
+### What did the client want
+
+The kickstarters were supposed to build an application that accepts incoming data, while being able to process this data and output it in a more user friendly way.
+The data would be send by 3 different sensors.
+
+The sensors that are used:
+   * Temperature sensor
+   * Humidity sensor
+   * Motion sensor
+
+These sensors sends some data every few seconds to the back-end, where this data is being processed and send back to the front-end.
+Here the front-end developers made sure all the data has been received and outputted in the correct way.
+
+The following picture shows us the 2 meeting rooms that are equiped with 3 different sensors, which send their data back to the application's back-end.
+
+<p style="text-align: center;">
+  <img alt="Floor Plan" src="/img/kickstarters/floorplan.jpg">
+</p>
+
+Each meeting room equiped with the sensors, which have their values read by an Arduino that then sends these across the Proximus LoRa network to the backend. For the initial stages and testing the LoRa part was omitted and a simple node server instance was used to relay the sensor values to the actual backend.
```

Comment:
backend is hier nu aan elkaar, overal consistent gebruiken (backend en frontend mogen aan elkaar imo)